### ISSUE TITLE
A11y_AzureDataStudio_BigDataClusterOverviewPage _ScreenReader : JAWS is announcning extra content that is "As Flows From" after the announcement of the each content present on the screen.

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/azuredatastudio#7672]
"[Check out Accessibility Insights!](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier."
​
GitHubTags:#A11y_ADS_OctTestPass;#A11yMAS;#A11yTCS;#MAS1.3.1;DesktopApp;#JAWS;#SQL AzureDataStudio;#Win32

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 6582debd7388b1be512fc3a9b956d3e54f6115e7
Date: 2019-10-03T08:56:27.454Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
JAWS Version:2019.1907.42 ILM

**Additional Details:**
Tool Name : JAWS

**Steps to Reproduce:**
1. Install and Launch the application the azure data studio application.
2. Open the “Connections” viewlet (CTRL + SHIFT + D)
3. Navigate to the “SQL Server Big Data Clusters” section and expand it
4. Navigate to the "Add SQL Server big data cluster controller…" node and click on it.
5. Fill in the connection information for the BDC Controller Endpoint
6. Click “Add”
7. Navigate to the controller node in the “SQL Server Big Data Clusters” view and right click on it, then select “Manage” to open the dashboard
8. "Big Data Cluster Overview page " should open.
9. Start JAWS and navigate using arrows.
10. Observe the announcement by screen reader .

**Actual:**
JAWS is announcing extra content that is "As Flows From" after the announcement of the content present on the screen.

**Similar issue is also observed with following screens:
1)Control tab**
1. Install and Launch the application the azure data studio application.
2. Open the “Connections” viewlet (CTRL + SHIFT + D)
3. Navigate to the “SQL Server Big Data Clusters” section and expand it
4. Navigate to the "Add SQL Server big data cluster controller…" node and click on it.
5. Fill in the connection information for the BDC Controller Endpoint
6. Click “Add”
7. Navigate to the controller node in the “SQL Server Big Data Clusters” view and right click on it, then select “Manage” to open the dashboard->click on the "control tab " in the left nav bar-> navigate to the "Metricsdc" tab.

**2)Jupyter books**
1. Install and Launch the application the azure data studio application.
2.Connet to the servers with the required credentials.
3.Navigate to the control panel and click on the "jupyter books" icon.
4.Press CRTL+SHIFT+P and search "jupyter books: SQL Server 2019 Guide" 

**Expected:**
JAWS should not announce  "As Flows From" after the announcement of the content present on the screen.

**User Impact:**
Screen reader user will get incorrect information regarding the content present on the screen.

**Attachment for Reference:**
[7672_A11y_AzureDataStudio_BigDataClusterOverviewPage _ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3717247/7672_A11y_AzureDataStudio_BigDataClusterOverviewPage._ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by Charles-Gagnon** [comment]
There are known issues with JAWS and so we (and VS Code) primarily support NVDA as the screen reader of choice. So closing this for now - let us know if this is a concern.

**Comment by 12-shweta** [comment]
Verified the bug in the latest version :
Version: 1.15.0-insider (user setup)
Commit: 56695be
Date: 2020-01-30T05:35:45.707Z
VS Code: 1.42.0
Electron: 6.1.6
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.18363
Issue is still reproducing .
Please find the attachment :
[7672_NotFixed.pptx](https://github.com/microsoft/azuredatastudio/files/4132987/7672_NotFixed.pptx)

**Comment by vinodbk197** [comment]
@Charles-Gagnon 
Verified this issue on :Version: 1.15.0-insider (user setup)
The issue is still reproduced, Please refer the above comments of @12-shweta. we are not able to reopen the bug. Kindly reactivate the bug as this bug is still reproduced

**Comment by Charles-Gagnon** [comment]
@12-shweta @vinodbk197 As my comment above mentions this is a known issue with JAWS. It does seem to reproduce when using NVDA so I'm closing this and will follow up offline.

**Comment by vinodbk197** [comment]
Verified this issue and since this issue works fine with one free screen reader.Hence this issue can be closed.

**Comment by fsteffi** [comment]
#Azure Data Studio - New Features_win32;

**Comment by Rama8345** [comment]
A11ySev3
